src/App.js

```diff
@@ -1,16 +1,15 @@
 import React, { Component } from 'react';
 import './App.css';
-import Header from './Components/Header';
-import Footer from './Components/Footer'
+import { Header, Footer } from './components';
 
 class App extends Component {
   render() {
     return (
       <div>
-      <div className="App">
+      <div className='App'>
```

Comment:
In jsx you should use double quote (className="App") and in raw js code you should use single quote ( var a = 'smth';)